codebuild/release/prod-release.yml

```diff
@@ -0,0 +1,28 @@
+version: 0.2
+
+env:
+  variables:
+    BRANCH: "master"
+  secrets-manager:
+    TWINE_USERNAME: PyPiAdmin:username 
+    TWINE_PASSWORD: PyPiAdmin:password
+
+phases:
+  install:
+    runtime-versions:
+      python: latest
+  build:
+    commands:
+      - pip install tox
+      - git checkout $BRANCH
+      - tox -e park
+      - tox -e release
+      - git clone https://github.com/aws-samples/busy-engineers-document-bucket.git
+      - cd busy-engineers-document-bucket/exercises/python/encryption-context-complete
+      - tox -e test
+
+
+batch:
```

Comment:
Why do these have to be batch builds?

Reply:
They don't need to be, this was mainly a) to facilitate my testing, where I was kicking off builds via the console (which uses batch builds) and b) future looking if we want to eventually take advantage of batch behavior.